lib/Formatters/TreduEventsFormatter.php

```diff
@@ -112,7 +112,8 @@ private function query_events( array $layout ) : array {
             'post_type'              => TreduEvent::SLUG,
             'update_post_meta_cache' => false,
             'no_found_rows'          => true,
-            'orderby'                => [ 'start_date' => 'ASC', 'title' => 'ASC' ],
+            'meta_key'               => 'start_date',
+            'orderby'                => [ 'meta_value_num' => 'ASC', 'title' => 'ASC' ],
```

Comment:
Luettavuuden ja koodin muokkauksen kannalta ois kivempi et ois rivitetty
ja pilkku pitäis lisää tohon vikan itemin perään linttereide mukaa.